app/src/test/java/com/example/beatboxapp/SoundViewModelTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.beatboxapp
+
+import org.hamcrest.CoreMatchers
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito
+
+class SoundViewModelTest {
+
+    private lateinit var beatBox: BeatBox
```

Comment:
You can annotate this with `@Mock` and you won't have to initialize it in `setup()`.
To use that annotation you'll also need to annotate the class with `@RunWith(MockitoJUnitRunner::class)`